app/src/main/java/com/duckduckgo/app/browser/WebViewRequestInterceptor.kt

```diff
@@ -223,6 +225,10 @@ class WebViewRequestInterceptor(
         url?.let { webViewClientListener?.onReceivedMaliciousSiteWarning(it, feed, exempted) }
     }
 
+    override fun addExemptedMaliciousSite(url: Uri, feed: Feed) {
+        maliciousSiteBlockerWebViewIntegration.onSiteExempted(url, feed)
```

Comment:
Posted on another PR (so depends on that answer), I assume we need to pass Feed to exempted logic.

Reply:
Yes, so that when a URL is identified as exempted, we can setup the privacy dashboard accordingly